src/common/components/kbase_icons.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: "kbase-icons";
+  src:url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.eot");
+  src:url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.eot?#iefix") format("embedded-opentype"),
+      url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.woff") format("woff"),
+      url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.ttf") format("truetype"),
+      url("https://ci.kbase.us/ui-assets/fonts/v1/kbase-icons.svg#kbase-icons") format("svg");
```

Comment:
do we know how https://github.com/kbase/ui-assets is deployed? Should these files instead be in `/public`? 

this is probably fine for now, but I imagine we'll want to merge ui-assets into europa asap? @charleshtrenholm @dakotablair thoughts on that?